File: cakk-api/src/main/java/com/cakk/api/dto/response/cake/CakeDetailResponse.java
package com.cakk.api.dto.response.cake;

import java.util.Set;

import lombok.Builder;

import com.cakk.common.enums.CakeDesignCategory;
import com.cakk.domain.mysql.dto.param.tag.TagParam;


@Builder
public record CakeDetailResponse(
	String cakeImageUrl,
	String cakeShopName,
	String shopBio,
	Long cakeShopId,
	Set<CakeDesignCategory> cakeCategories,
	Set<TagParam> tags
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/cake/CakeDetailResponse.kt
package com.cakk.core.dto.response.cake

import com.cakk.common.enums.CakeDesignCategory
import com.cakk.infrastructure.persistence.param.tag.TagParam

data class CakeDetailResponse(
    val cakeImageUrl: String,
    val cakeShopName: String,
    val shopBio: String,
    val cakeShopId: Long,
    val cakeCategories: Set<CakeDesignCategory>,
    val tags: Set<com.cakk.infrastructure.persistence.param.tag.TagParam>
)
